lib/cc/analyzer/issue_path_is_file_validation.rb

```diff
@@ -0,0 +1,19 @@
+module CC
+  module Analyzer
+    class IssuePathIsFileValidation < Validation
+      def valid?
+        File.file?(path)
```

Comment:
This is `false` for symlinks. That's a kind of significant product change that should probably be discussed.